### ISSUE TITLE
Selenium-C#- V 4.0.0-rc1-DesiredCapabilities has been deprecated and henceforth the WinAppDriver is not working

### ISSUE THREAD

**Issue by svadivel0006** [microsoft/WinAppDriver#1590]
Selenium-C#- V 4.0.0-rc1-DesiredCapabilities has been deprecated and henceforth the WinAppDriver is not working

Expected behavior:

DesiredCapabilities should not be deprecated, as it required for automating the WinAppDriver

Environment

OS: Windows 10
WinAppDriver version: WinAppDriver v1.3 Release Candidate 1 (1.2.99)
Language Bindings version: C#
Selenium version : Selenium-C#- V 4.0.0-rc1

![DesiredCapabilities](https://user-images.githubusercontent.com/85741928/133009704-a5647e51-a8fa-45ab-ade4-782ea1ff35fc.png)

![DesiredCapabilities1](https://user-images.githubusercontent.com/85741928/133009708-c61c51f3-e25b-4b29-9a63-14a3bc3ef3f5.png)

**Comment by anunay1** [comment]
Duplicate of this [Bug](https://github.com/appium/appium-dotnet-driver/issues/460)

**Comment by ShanmugaPraveenG** [comment]
@svadivel0006 Do we have have any alternative for DesiredCapabilities in 4.0 which can be used along with WinAppDriver to automate windows app

**Comment by anunay1** [comment]
AppiumOptions

**Comment by dtopuzov** [comment]
WinAppDriver can not be used with `Appium.WebDriver@5.0.0-beta01` too:
https://github.com/appium/appium-dotnet-driver/issues/487

If you want to use latest appium in order to test your Android and iOS apps you should use `Appium.WebDriver@5` because at some point Appium 1.x will be [end of life](https://github.com/appium/appium/blob/2.0/docs/en/about-appium/1.x-eol.md).
 but in order to use appium to test Windows desktop apps you should use ``Appium.WebDriver@4`.
This makes it imposible to build testing framework for [.NET MAUI](https://docs.microsoft.com/en-us/dotnet/maui/what-is-maui) apps were you test Android, iOS, Windows and macOS with single code base.

**Comment by diptilenka** [comment]
I am also facing the same issue @anunay1 . what is the solution here.

![Screenshot 2023-01-25 213206](https://user-images.githubusercontent.com/63293480/214613425-6725da9c-78b0-44f4-893f-219bec1c00e8.png)

**Comment by anunay1** [comment]
Degrade the selenium version to 3.

**Comment by Shakevg** [comment]
[diptilenka](https://github.com/diptilenka) https://learn.microsoft.com/en-us/nuget/consume-packages/install-use-packages-visual-studio

Select the previous version of Selenium 3.14 and click install

**Comment by diptilenka** [comment]
> [diptilenka](https://github.com/diptilenka) https://learn.microsoft.com/en-us/nuget/consume-packages/install-use-packages-visual-studio
> 
> Select the previous version of Selenium 3.14 and click install



> [diptilenka](https://github.com/diptilenka) https://learn.microsoft.com/en-us/nuget/consume-packages/install-use-packages-visual-studio
> 
> Select the previous version of Selenium 3.14 and click install



> [diptilenka](https://github.com/diptilenka) https://learn.microsoft.com/en-us/nuget/consume-packages/install-use-packages-visual-studio
> 
> Select the previous version of Selenium 3.14 and click install
![Screenshot 2023-01-26 132851](https://user-images.githubusercontent.com/63293480/214784723-f458d64a-76f9-4797-90a2-bcc5aff0845a.png)
![Screenshot 2023-01-26 132851](https://user-images.githubusercontent.com/63293480/214784723-f458d64a-76f9-4797-90a2-bcc5aff0845a.png)
![Screenshot 2023-01-26 132851](https://user-images.githubusercontent.com/63293480/214784723-f458d64a-76f9-4797-90a2-bcc5aff0845a.png)
![Screenshot 2023-01-26 132851](https://user-images.githubusercontent.com/63293480/214784750-6fb32991-6468-48f2-979a-bf56bd53f67f.png)
![Screenshot 2023-01-26 132851](https://user-images.githubusercontent.com/63293480/214784750-6fb32991-6468-48f2-979a-bf56bd53f67f.png)
![Screenshot 2023-01-26 132851](https://user-images.githubusercontent.com/63293480/214784750-6fb32991-6468-48f2-979a-bf56bd53f67f.png)

**Comment by diptilenka** [comment]
Already done but still Desised capabilities shows error, by the way i got resolved those using 
` var appCapabilities = new AppiumOptions();
                appCapabilities.AddAdditionalCapability("app", calApp);
                appCapabilities.AddAdditionalCapability("platformName", "Windows");`

**Comment by Shakevg** [comment]
@diptilenka Yes, you are right if you use Appium.WebDriver you should use AppiumOptions. Sorry, miss this issue.
 https://github.com/microsoft/WinAppDriver/blob/a6f3d7bca447e0c13a33f76454600c1fe737ab10/Docs/AuthoringTestScripts.md#testing-a-classic-windows-application
